cryptofeed/backends/arctic.py

```diff
@@ -33,9 +33,10 @@ def __init__(self, library, host='127.0.0.1', key=None, numeric_type=float, **kw
         self.key = key if key else self.default_key
         self.numeric_type = numeric_type
 
-    async def write(self, feed, pair, timestamp, data):
+    async def write(self, feed, pair, timestamp, receipt_timestamp, data):
         df = pd.DataFrame({key: [value] for key, value in data.items()})
         df['date'] = pd.to_datetime(df.timestamp, unit='s')
+        df['receipt_timestamp'] = pd.to_datetime(df.receipt_timestamp, unit='s')
```

Comment:
is seconds granular enough here, or should it be 'ms'

Reply:
if you have receipt_timestamp explicitly passed, it probably makes more sense to use that rather than getting it from the data object.

I have another comment further down about passing receipt_timestamp explicitly though.

Reply:
you're telling pandas what the data its formatting is in, the data is unix epoch time, so its in seconds, hence the 's'.